### ISSUE TITLE
Proposing removal of non-active California chapters

### ISSUE THREAD

**Issue by FrancesCoronel** [nodeschool/organizers#513]
Some California chapters have not had their websites or associated GitHub repos updated in a **long** time and I think it would be in everyone's best interest to remove these chapters from the Node School website: https://nodeschool.io/chapters.html.

Having inactive chapters listed leads to loss of time for an interested user as they try and dig around to see if the chapter is still alive.

Ideally, all the chapters are reviewed at least bi-yearly to check if they are still active.

A good example of why I think this is important can be shown in the GitHub issue that was raised for the San Luis Obispo chapter: https://github.com/nodeschool/san-luis-obispo/issues/4

It seems the following chapters are no longer active and have not held new events for a while:

- Mountain View - https://github.com/nodeschool/baynode
- Modesto - https://nodeschool.io/modesto/
- Orange County - https://github.com/nodeschool/oc
- San Luis Obispo - https://nodeschool.io/san-luis-obispo/

**Comment by riyadhalnur** [comment]
I agree but at the same time, in my opinion, I feel chapter removal requests should go through the organisers of the particular chapter. Once approved, we can go ahead and remove them. Any other thoughts are welcome.

**Comment by fvcproductions** [comment]
@riyadhalnur Makes sense. I went ahead and opened an issue of their respective GitHub repos to see what the organizers think. Some GitHub repos have been inactive for 2 years+ so fingers crossed. 🤞️
